src/modules/__test__/ItemsCounter.test.js

```diff
@@ -0,0 +1,8 @@
+import ItemsCounter from '../ItemsCounter.js';
```

Comment:
- [ ] Good Job testing all of the functions of the counters 🎉 but please note that you should test and cover all edge cases that can happen, I would suggest testing the case of having 0 items in the array where the expected result should be 0, by implementing that you will cover all of the edge cases 🙏 kindly fix that
- [ ] kindly try to use a correct descriptive message in your describe() method, in [ItemsCounter.test.js](https://github.com/Baayeh/second-capstone/pull/36/files#diff-c43a186005a06939bfe9495b50c3daa760cc2bc5d5c867440015c09b3a4f32ba) you described the test as `CommentsCounter` but it's not a comment counter , kindly fix it